doc/source/examples/incompressible-flow/2d-mms/2d-mms.rst

```diff
@@ -0,0 +1,251 @@
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+The velocity components are set to zero on all of the boundary to match the analytical solution. However, the condition on each boundary can be changed depending on the analytical function used.
+
+.. code-block:: text
+
+  subsection boundary conditions
+  set number = 1
+    subsection bc 0
+      set id = 0
+      set type = noslip
+    end
+  end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+For this analysis, the Reynolds number (based on the maximum velocity in the domain) is taken to be 1, which results in the terms of the NS equation being of the same order. Hence, the kinematic viscosity is set to 1 in the ``physical properties`` subsection:
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 1
+      end
+    end
+
+
+FEM Interpolation
+~~~~~~~~~~~~~~~~~
+
+Similarly to the ``initial refinement`` parameter in the case of a simplex mesh, ``velocity order`` and ``pressure order``  are set to the {{Poly_deg_u}} and {{Poly_deg_p}} parameter variables, respectively, in Jinja2 format.
+
+.. code-block:: text
+
+  subsection FEM
+    set velocity order = {{Poly_deg_u}}
+    set pressure order = {{Poly_deg_p}}
+  end
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+The nonlinear solver's tolerance is set to :math:`10^{-10}` since the errors on the velocity and pressure fields reach values that are near or lower than the default tolerance of :math:`10^{-8}` on the quadrilateral meshes with a finer resolution. Hence, if the default tolerance were used, the error would be constrained by this value.
+
+.. code-block:: text
+
+    subsection non-linear solver
+      subsection fluid dynamics
+        set tolerance = 1e-10
+        set verbosity = verbose
+      end
+    end
+
+
+Linear Solver
+~~~~~~~~~~~~~
+
+The only modification made in the linear solver section is the use of AMG preconditioner to accelerate the simulations. 
+
+.. code-block:: text
+
+  subsection linear solver
+    subsection fluid dynamics
+      # Set type of preconditioner for the iterative solver
+      set preconditioner = amg
+      set verbosity = verbose
+    end
+  end
+
+-----------------------
+Running the Simulations
+-----------------------
+
+
+The simulations are launched by first running the ``generate_cases.py``, then ``launch_cases.py`` scripts. The first script generates the folders and parameter files for the different configurations simulated,
```

Comment:
It might be a good idea to have the number of cores to use as an argument of the launch_cases.py
Furthermore, when you launch the script it is not giving any indication of which job is currently being launched. I would just have something that indicates which simulation is running, otherwise this is very confusing for the person running the script. Maybe it would be good to just display the logs of the simulation so someone can know what is going on. Otherwise I was confused when running the script (is it running, etc.)